test/evergreen.yml

```diff
@@ -1026,6 +1026,22 @@ tasks:
             pip install scons==3.1.1
             scons-3.1.1.bat ${smp_command|} "CFLAGS=/Gv /wd4090 /wd4996 /we4047 /we4024 /TC /we4100 /we4133" wiredtiger.dll libwiredtiger.lib
 
+  - name: scons-check
+    tags: ["pull_request", "windows_only"]
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            pip install scons==3.1.1
+            scons-3.1.1.bat ${smp_command|} check
```

Comment:
Is `${smp_command}` actually blank for the Windows build variant? I can see the compile task uses the same variable, but am not aware of any Windows alternative to `cat /proc/cpuinfo`.